README.md

```diff
@@ -129,6 +129,20 @@ MessageBus.publish "/channel", "hello", client_ids: ["XXX", "YYY"], user_ids: [1
 
 Passing `nil` or `[]` to either `client_ids`, `user_ids` or `group_ids` is equivalent to allowing all values on each option.
 
+### Filtering Messages
+
+Custom message filters can be registered via `MessageBus#register_message_filter`. This can be useful for filtering away messages from the client based on the message's payload.
+
+For example we can ensure that only messages seen by the server in the last 20 seconds are published to the client:
+
+```
+MessageBus.register_message_filter(/^\/test\/\d+$/) do |message|
```

Comment:
let's call it `register_client_message_filter` ... since it only applies to clients and does not apply to stuff like just calling backlog or subscribe.